src/Hyperrectangle.jl

```diff
@@ -177,3 +177,37 @@ A real number representing the diameter.
 function diameter(H::Hyperrectangle, p::Real=Inf)
     return 2. * radius(H, p)
 end
+
+"""
+    high(H::Hyperrectangle)
+
+Return the higher coordinates of a hyperrectangle.
+
+### Input
+
+- `H` -- a hyperrectangle
+
+### Output
+
+A vector with the higher coordinates of the hyperrectangle, one entry per dimension.
+"""
+function high(H::Hyperrectangle)
+    return H.center + H.radius
```

Comment:
Can you do `.+` here? Also below...
Not tested...

Reply:
yep, i did it and added tests. thanks :)